services/Amazon.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Amazon",
+    "documents": {
+        "Terms of Service": {
+            "fetch": "https://www.amazon.fr/gp/help/customer/display.html/ref=hp_left_v4_sib?ie=UTF8&nodeId=201909000",

```

Comment:
For Amazon, this would be https://www.amazon.com/gp/help/customer/display.html?nodeId=GLSBYFE9MGKKQXXM.

We can also track https://www.amazon.com/gp/help/customer/display.html?nodeId=GX7NJQ4ZB8MHFRNJ (Privacy Policy type).

There is also :
* https://www.amazon.com/gp/help/customer/display.html?nodeId=202119380 which I guess would be "Commercial Terms",
* https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=GFUSWPZP368LFFA9 which would be "Acceptable Use Policy".

https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=GUKKCM63WPRVL86G is of high interest but does not yet have a matching document type in CGU.

Same for https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=GEVHX6WVCYZHQ67A / https://www.amazon.com/gp/help/customer/display.html?ie=UTF8&nodeId=GZ8H2UM2BWKDCCLQ / https://www.amazon.com/gp/help/customer/display.html/ref=hp_bc_nav?ie=UTF8&nodeId=202101440. Small interest for these latter ones I think.